deployment/ccip/changeset/v1_5_1/cs_add_token_e2e.go

```diff
@@ -187,6 +189,12 @@ type DeployTokenConfig struct {
 	// MintTokenForRecipients is a map of recipient address to amount to be transferred or minted
 	// and provided the minting role after token deployment.
 	MintTokenForRecipients map[common.Address]*big.Int `json:"mintTokenForRecipients,omitempty"`
+
+	// PreMint is the amount of tokens to pre-mint for the token.
+	PreMint *big.Int `json:"preMint,omitempty"`
+
+	// CCIPAdmin is the address of the CCIP admin for the token and will have default admin role.
+	CCIPAdmin common.Address `json:"ccipAdmin,omitempty"`
```

Comment:
minor nit: ill add a comment saying these new variables are applicable only for the ERC20BnM